scripts/Gripper.py

```diff
@@ -58,10 +61,15 @@ def __init__(self, config: GripperConfig):
 
         self.servos = {}
         self.target_servo = None
+        self.actuated_target = config.actuated_target
 
-        if config.actuated_target:
+        if config.actuated_target is not None:
```

Comment:
As we move to the object not being a servo it should be moved out into the environment and not part of the gripper class.